CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.5.2
```

Comment:
We cut releases as a stand-alone process, but we can def start fleshing out the release notes in the meantime. Please change this version number to “next” for now.

Reply:
Done.